### PR TITLE
Can now use a default value when replacing a JSONPath

### PR DESCRIPTION
**Pull request by Elyrixia**

### CODE REVIEW

**Comment by Elyrixia** [comment]
Need to update README

File: README.md [hunk 1]
```diff
@@ -47,6 +47,34 @@ HTTP/1.1 200 OK
 I found 12 for $.value. But nothing for ${$.undefined}
 ```
 
+You can check every supported operators on the [Gatling JSONPath Syntax](https://github.com/gatling/jsonpath#syntax) documentation.  
+
+In addition to this syntax, you can specify a default value if nothing was found for a specific JSONPath.
```
Comment (AlixBa) [inline README.md]: Maybe we could do only one example with all the "features".
Take the one above and make another sentence with "But I can put ${$.undefined§default} instead.

File: src/main/scala/tv/teads/wiremock/extension/JsonExtractor.scala [hunk 2]
```diff
@@ -17,9 +17,11 @@ class JsonExtractor extends ResponseTransformer {
 
   override val applyGlobally: Boolean = false
 
-  val pattern: Regex = """\$\{(\$\.[ ='a-zA-Z0-9\@\.\[\]\*\,\:\?\(\)]*)\}""".r
+  private val defaultR: Regex = """(?:\§(.+))""".r
```
Comment (AlixBa) [inline src/main/scala/tv/teads/wiremock/extension/JsonExtractor.scala]: i would have gone for the full "regex" suffix :p

File: src/main/scala/tv/teads/wiremock/extension/JsonExtractor.scala [hunk 3]
```diff
@@ -63,9 +65,12 @@ class JsonExtractor extends ResponseTransformer {
         val path: String = matched.group(1)
         val toAdd: String = extractValue(requestBody, path) match {
           case None ⇒
-            // since we don't have anything to replace
-            // we will add the raw template to the output body
-            template.take(matched.end)
+            // If there is a default value, use it
+            // else just keep the raw template
+            matched.group(2) match {
```
Comment (AlixBa) [inline src/main/scala/tv/teads/wiremock/extension/JsonExtractor.scala]: Option(matched.group(2)) to be scala idiomatic?

**Pull request merged**